RefRed/interfaces/metadata_finder_interface.ui

```diff
@@ -265,7 +265,7 @@
    <sender>unselectAll</sender>
    <signal>clicked()</signal>
    <receiver>MainWindow</receiver>
-   <slot>unselectAll()</slot>
+   <slot>unselectAllClicked()</slot>
```

Comment:
These names were causing issues when used by `loadUI` because there were widgets with the same names that created conflicts.

You can't have  `<sender>unselectAll</sender>` and `<slot>unselectAll()</slot>` with the same name.